src/index.js

```diff
@@ -0,0 +1,142 @@
+    let currentLandscape = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+
+    if (currentTemperature <= 49) {
+        currentColor = '#30849D';
+    } else if (currentTemperature >= 50 && currentTemperature <= 59 ) {
+        currentColor = '#50A63F';
+    } else if (currentTemperature >= 60 && currentTemperature <= 69 ) {
+        currentColor = '#FEE086';
+        currentLandscape = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+    } else if (currentTemperature >= 70 && currentTemperature <= 79 ) {
+        currentColor = '#F8B5A2';
+        currentLandscape = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+    } else if (currentTemperature >= 80) {
+        currentColor = '#E06767';
+        currentLandscape = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+    }
+    BackgroundColor.style.backgroundColor = currentColor;
+    elementFontColor.style.color = currentColor;
+    elementLandscape.textContent = currentLandscape;
+};
+
+const citySky = (event) => {
+    const selectedOption = event.target.value;
+    const elementSky = document.getElementById('sky');
+    let currentCitySky = '';
+
+    if (selectedOption === 'sunny') {
+        currentCitySky = '☀️ ☁️ ☀️ ☀️ ☁️ ☀️ ☁️ ☀️ ☀️ ☁️ ☀️';
+    } else if (selectedOption === 'cloudy') {
+        currentCitySky = '☁️☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️☁️';
+    } else if (selectedOption === 'rainy') {
+        currentCitySky = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧';
+    } else if (selectedOption === 'snowy') {
+        currentCitySky = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+    }
+    elementSky.textContent = currentCitySky;
+};
+
+const cityNameInput = document.getElementById('cityNameInput');
+const cityHeaderElement = document.getElementById('defaultHeaderCityName');
+
+const dynamicCityNameHeader = () => {
+    cityHeaderElement.innerText = cityNameInput.value;
+};
+
+const resetCityName = () => {
+    cityNameInput.value = ''; 
+    cityHeaderElement.innerText = '🌸🗼🍜🍣🌋🐬🌊🐢🌺🍍🌴🗽🍕🗼🥐';
+};
+
+const fetchCityTemperature = () => {
+    const cityName = cityNameInput.value.trim(); 
+
+    if (cityName) { 
+        console.log(`Fetching city: ${cityName}`);
+        axios.get(`http://127.0.0.1:5000/location`, {
+            params: {
+                q: cityName
+            }
+        })
+        
+        .then(response => {
+            console.log('Location Data:', response);
+                const firstResult = response.data[0];  
+                const latitude = firstResult.lat;
+                const longitude = firstResult.lon;
+                console.log(`Coordinates: lat=${latitude}, lon=${longitude}`);
+        
+                return axios.get(`http://127.0.0.1:5000/weather`, {
+                    params:{
+                        lat: latitude,
+                        lon: longitude
+                    }
+                })
+        })
+            .then(weatherResponse => {
+                const firstWeatherResult = weatherResponse.data.main.temp;
+                const kelvinToFahrenheit = (firstWeatherResult - 275.15) * 1.8 + 32;
+                 
+                const cityTemp = Math.floor(kelvinToFahrenheit); 
```

Comment:
Prefer this logic is in a helper function. I know it's not much code, but it would make this logic here concerned with one less thing and a descriptively named function name would make your code more self-documenting.